src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs

```diff
@@ -67,26 +175,35 @@ internal set
             }
         }
 
+        public virtual string Path
+        {
+            get { return this.entityPath; }
```

Comment:
>entityPath [](start = 30, length = 10)

you dont need this. you could convert this to autoProperty with get and private set